engine/swarm/admin_guide.md

```diff
@@ -251,8 +237,68 @@ Before you forcefully remove a manager node, you must first demote it to the
 worker role. Make sure that you always have an odd number of manager nodes if
 you demote or remove a manager
 
+## Back up the swarm
+
+Docker manager nodes store the swarm state and manager logs in the
+`/var/lib/docker/swarm/raft` directory. In 1.13 and higher, the Raft data is
+encrypted, and the encryption keys are stored in
+`/var/lib/docker/swarm/certificates`. Without these keys, you will not be able
+to restore the swarm.
+
+You can back up the swarm using any manager. Use the following procedure.
+
+1.  Stop Docker on the manager before backing up the data, so that no data is
+    being changed during the backup. It is possible to take a backup while the
+    manager is running (a "hot" backup), but this is not recommended and your
+    results will be less predictable when restoring. While the manager is down,
+    other nodes will continue generating swarm data that will not be part of
+    this backup.
+
+    > **Note**: Be sure to maintain the quorum of swarm managers. For instance,
+    > if you normally run with three managers, you need to add a manager before
+    > you can stop one, because two managers are not sufficient to maintain the
+    > quorum.
+
+2.  Back up the `raft` directory, and in 1.13 and higher, the `certificates`
+    directory, using operating system utilities.
+
+3.  Restart the manager.
+
+To restore, see [Restore from a backup](#restore-from-a-backup).
+
 ## Recover from disaster
 
+### Restore from a backup
+
+After backing up the swarm as described in
+[Backing up the swarm](#backing-up-the-swarm), use the following procedure to
+restore the data to a new swarm.
+
+1.  Initialize a new single-node swarm.
+
+2.  Shut down Docker on that node.
+
+3.  Remove the contents of `/var/lib/docker/swarm/raft` directory on the new
+    swarm, and on 1.13 and higher, the `/var/lib/docker/swarm/certificates`
+    directory.
+
+4.  Restore the `/var/lib/docker/swarm/raft` directory with the contents of the
+    backup. In Engine 1.13 and higher, also restore the
+    `/var/lib/docker/swarm/certificates` directory.
+
+    > **Note**: The new swarm will use the same encryption keys as the old one.
+    > It is not possible to change the encryption keys on a swarm at this time.
```

Comment:
"The new node will use the same encryption key for on-disk storage as the old one. It is not possible to change the on-disk storage encryption keys at this time."